src/components/RichNote.tsx

```diff
@@ -41,19 +41,15 @@ export default function RichNote({
 
       {formattedMediaLinks?.length ? (
         <div style={{ display: 'block' }}>
-          <Space size="middle" direction="vertical">
-            {formattedMediaLinks.map((links, i) => (
-              <Space key={i} size="middle">
-                {links.map((link, i) => (
-                  <RichImgPreview
-                    key={i}
-                    src={link}
-                    style={{ marginTop: '0.5rem', padding: '0.5rem' }}
-                    width="100%"
-                    height="6rem"
-                  />
-                ))}
-              </Space>
+          <Space size="middle">
+            {formattedMediaLinks.map((link, i) => (
```

Comment:
So the reason for the above was to allow the ability to have as many images as you want, but display them per line (with a max of 3 per line). I’m not sure we want to undo this right?

Reply:
Reasoning in this comment: https://github.com/jbx-protocol/juice-interface/pull/1520#issuecomment-1198717704. 

I think there is an issue in letting people show as many images as they want - people would've been abusing it harrrd during Assange/Constitution and it would've looked quite bad having some pay items with 10+ images promoting some random NFT